MvvmCross/Platforms/Wpf/Views/MvxWpfView.cs

```diff
@@ -23,6 +24,8 @@ public IMvxViewModel ViewModel
             }
         }
 
+        public IMvxBindingContext BindingContext { get; set; }
```

Comment:
Looks like implementation is missing. Other views instantiate it here usually.

```suggestion
        private IMvxBindingContext _bindingContext;
        public IMvxBindingContext BindingContext
        {
            get
            {
                if (_bindingContext == null)
                    _bindingContext = new MvxBindingContext();
                return _bindingContext;
            }
            set
            {
                _bindingContext = value;
            }
        }
```